ts/src/okx.ts

```diff
@@ -2290,20 +2290,12 @@ export default class okx extends Exchange {
         return this.filterBySymbolSinceLimit (sorted, market['symbol'], since, limit) as FundingRateHistory[];
     }
 
-    parseBalanceByType (type, response) {
```

Comment:
why are we removing this?

Reply:
I don't even know how the build is working without this method 🤔

Reply:
With no reason. It was my deleting? If it was it is just a typo, although I don't remember removing this method

Reply:
I add it